tests/test_app.py

```diff
@@ -307,6 +325,15 @@ def test_title_search_plain_text_value_format(self, mock_get):
         )
         assert '0 results found' in response.data.decode()
 
+    @mock.patch('service.auditing.audit')
+    @mock.patch('requests.get', return_value=fake_no_titles)
+    def test_title_search_audits_the_events(self, mock_get, mock_audit):
+        search_term = 'search term'
+        self.app.post('/title-search', data=dict(search_term=search_term), follow_redirects=True)
```

Comment:
Since you're not doing anything dynamic here, I think it would be clearer to use {'search_term': search_term} instead of using dict().